File: SDKs/twilio.com/twilio_insights_v1/1.37.4/java/lib/src/main/java/openapisdk/models/shared/CallSummariesEnumCallStateEnum.java
package openapisdk.models.shared;


public enum CallSummariesEnumCallStateEnum {
    RINGING("ringing"),
    COMPLETED("completed"),
    BUSY("busy"),
    FAIL("fail"),
    NOANSWER("noanswer"),
    CANCELED("canceled"),
    ANSWERED("answered"),
    UNDIALED("undialed");

    public final String value;

    private CallSummariesEnumCallStateEnum(String value) {
        this.value = value;
    }
}

File: SDKs/twilio.com/twilio_insights_v1/1.37.4/python/src/sdk/models/shared/call_summaries_enum_call_state_enum.py
import dataclasses
from enum import Enum

class CallSummariesEnumCallStateEnum(str, Enum):
    RINGING = "ringing"
    COMPLETED = "completed"
    BUSY = "busy"
    FAIL = "fail"
    NOANSWER = "noanswer"
    CANCELED = "canceled"
    ANSWERED = "answered"
    UNDIALED = "undialed"

